Adding an example for extending a timeout for pipeline (#867)
packit/packit.dev: docs/configuration/examples.md

</details>

<details>

>   <summary>Extending timeout of pipeline for 24 hours (default is 12 hours)</summary>

Reviewer: ```suggestion
  <summary>Extending timeout of a test execution to 24 hours (default is 12 hours)</summary>
```

just so that it doesn't clash with Packit's pipeline terminology (see https://dashboard.packit.dev/pipelines)

Author: Yeah sure 💯 Thanks :)
